### PR TITLE
la-pipelines CLI

### PR DESCRIPTION
**Pull request by vjrj**
This PR includes several works, discussed in AtlasOfLivingAustralia/la-pipelines#88 like:
- Initial `la-pipelines` that deprecated many of  `livingatlas/scripts` 
- colorize logs
- `la-pipelines` CLI bash autocompletion

It is also a continuation of: 
https://github.com/AtlasOfLivingAustralia/la-pipelines/pull/90

and should be continued by a `la-pipelines` debian package.

More details in my today presentation to ALA team:
https://demo.gbif.es/local-files/la-pipelines-cli-2020.html#1

### CODE REVIEW

File: livingatlas/pipelines/pom.xml [hunk 1]
```diff
@@ -603,6 +603,11 @@
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.jcabi</groupId>
+      <artifactId>jcabi-log</artifactId>
+      <version>0.18.1</version>
```
Comment (timrobertson100) [inline livingatlas/pipelines/pom.xml:609]: Nit: suggest keep versions in properties

File: livingatlas/scripts/la-pipelines [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+  --extra-args=<args>  Additional "arg1=values,arg2=value" to pass to pipeline options (highest precedence than yml values).
+  --no-colors          No colorize logs output.
+  --dry-run            Print the commands without actually running them.
+  --debug              Debug $CMD.
+  -h --help            Show this help.
+  -v --version         Show version.
+----
+$CMD $VER
+Copyright (C) 2020 ALA Development Team
+License MPL v 1.1
```
Comment (MattBlissett) [inline livingatlas/scripts/la-pipelines:82]: I notice ALA normally uses a different license (MPL) than GBIF (Apache).  We should probably have one license for the whole project.

File: livingatlas/pipelines/pom.xml (hunk 1 above)
Comment (djtfmartin) [reply livingatlas/pipelines/pom.xml:609]: I've fixed this post-merge.

File: livingatlas/scripts/la-pipelines (hunk 2 above)
Comment (djtfmartin) [reply livingatlas/scripts/la-pipelines:82]: Apologies - I was looking at a stale version of the PR page and missed this comment.  I think we should remove the Copyright line and change the licence to Apache to comply with the existing licence.

Comment (vjrj) [reply livingatlas/scripts/la-pipelines:82]: Sorry, my fault. I started this CLI prior to the GBIF merge and I use the MPL license here and there. @djtfmartin forget this.

Comment (djtfmartin) [reply livingatlas/scripts/la-pipelines:82]: thanks @vjrj

**Pull request merged**